### DISCUSSION TITLE
Config question for trailingstoploss_tp.py

### DISCUSSION THREAD

**Discussion by ifmvaiea** [cyberjunky/3commas-cyber-bots#201] [General]
Hello,

For the Safety Config, after we have disabled Max safety orders count, will the script continue to respect the **Safety order volume scale** settings for each successive safety order?

Thank you!

**Comment by amargedon** [comment]
I would think so. Not sure what you mean by the disabled setting, as the max number is required for the script. Did you mean the number of active safety orders set to zero?
